src/components/page-size/OcPageSize.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <label class="oc-page-size">
```

Comment:
I think we agreed on putting the `<label>` next to input fields and connecting them via `for/id` rather than wrapping them around but not sure how strict we want to be with this

Reply:
Is that documented somewhere? Don't really understand why we'd want to block the easier way to do this...